src/GitList/Git/Client.php

```diff
@@ -53,8 +53,14 @@ public function getRepositories($paths)
         return $allRepositories;
     }
 
-    private function recurseDirectory($path, $topLevel = true)
+    private function recurseDirectory($path, $topLevel = true, $basePath = "")
     {
+        if ($topLevel) {
+            $basePath = realpath($path) . DIRECTORY_SEPARATOR;
```

Comment:
Calling realpath($path) breaks substitution if symlink is configure as repository in config.ini. Also you should check it $path doesn't already ends with DIRECTORY_SEPARATOR (if it does, it gets duplicated).